config.example.yml

```diff
@@ -11,8 +11,10 @@ cpus: 2
 # - shortname: The site's shortname.
```

Comment:
Let's chuck it into the same config but all commented out. People will need to read or they'll get yelled at. :smile: